docker/Dockerfile.jetson

```diff
@@ -47,7 +47,8 @@ RUN --mount=target=/var/lib/apt/lists,type=cache,sharing=locked \
                            # imagemagick with c++ dependency
                            libmagick++-dev \
                            # needed for pytorch 
-                           libopenblas-dev
+                           libopenblas-dev \
+                           ninja-build
```

Comment:
How was this missing before

Reply:
:shrug: maybe we just apt installed it everytime lmao